tests/src/tests/cluster.rs

```diff
@@ -0,0 +1,181 @@
+use ckb_testtool::ckb_types::H256;
+use ckb_testtool::ckb_types::{core::TransactionBuilder, packed, prelude::*};
+use ckb_testtool::context::Context;
+
+use crate::utils::co_build::*;
+use crate::utils::*;
+use crate::MAX_CYCLES;
+
+#[test]
+fn test_simple_cluster_mint() {
+    let mut context = Context::default();
+
+    let cluster = build_serialized_cluster_data("Spore Cluster", "Test Cluster");
+    let (cluster_out_point, cluster_script_dep) = build_spore_materials(&mut context, "cluster");
+    let input_cell = build_normal_input(&mut context);
+    let cluster_type_id = build_type_id(&input_cell, 0);
+    let type_ = build_spore_type_script(
+        &mut context,
+        &cluster_out_point,
+        cluster_type_id.to_vec().into(),
+    );
+    let cluster_out_cell = build_output_cell_with_type_id(&mut context, type_.clone());
+
+    let tx = TransactionBuilder::default()
+        .input(input_cell)
+        .output(cluster_out_cell)
+        .output_data(cluster.as_slice().pack())
+        .cell_dep(cluster_script_dep)
+        .build();
+
+    let action = build_cluster_create_action(&mut context, cluster_type_id, cluster.as_slice());
+    let tx = complete_co_build_message_with_actions(tx, &[(type_, action)]);
+
+    let tx = context.complete_tx(tx);
+
+    context
+        .verify_tx(&tx, MAX_CYCLES)
+        .expect("test simple spore mint");
+}
+
+#[test]
+fn test_simple_spore_mint_with_cluster() {
+    let cluster_id = H256::from_trimmed_str("12345678".clone())
```

Comment:
Why is cluster_id defined as `from_trimmed_str("12345678"`?

### ref
CLUSTER_ID = `hash(this_transaction.inputs[0]) | Grouputput_index_of_this_cell`